tools/mo/openvino/tools/mo/utils/check_ie_bindings.py

```diff
@@ -70,7 +70,10 @@ def import_core_modules(silent: bool, path_to_module: str):
         print("{}: \t{}".format("Model Optimizer version", mo_version))
 
```

Comment:
Also, we can hardcode major and minor versions of MO for now, i.e.: 2022.1. And update v.get_version() function with this prefix

Reply:
hardcoded major and minor versions into MO